HHS/Head-Start-TTADP
# Changes to visual styles related to the "X selected" capsule widget and filter menus/Correct and bypass yarn audit vul

## Description of change
* Changes to visual styles related to the "X selected" capsule widget and filter menus. Also some a11y cleanup.
* Deploys via ci/cd pipeline are blocked by new npm package vulnerabilities detected by yarn audit (our static security scan). This upgrades some npm packages to include new patches with vulnerability fixes and ignores other vulnerabilities. Ignored vulnerabilities are all found in dependent subpackages, where the main package has not been updates to use latest version, so they can not be reasonably or safely upgraded. All ignored vulnerabilities are also devDependency packages so they are unlikely to pose a security list and can be safely ignored to allow regular development to continue.

## How to test
* Start with multiple approved reports. On the "Activity reports" table, select one or more reports using the checkboxes, and a blue capsule widget should appear. Check that the capsule aligns with the other menus, and that all menus are ~16px font-size, per the designs. All text in the menu areas should be sans-serif, whereas before the capsule widget was rendering with a serif font.

* Pipeline no longer fails, be and fe tests pass on circleCI 

## Issue(s)
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-147

## Checklists

### Every PR


~Code is meaningfully tested~
~API Documentation updated~
~Boundary diagram updated~
~Logical Data Model updated~
~[Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
